access-restrictor/lua/src/ratelimit.lua

```diff
@@ -21,7 +21,7 @@ end
 
 function RateLimiter:reject(time_remaining)
   ngx.header["Content-Type"] = "text/plain"
-  ngx.header["X-RateLimit:Limit"] = "1" -- maximum requests per time period
```

Comment:
Die Tests haben Probleme mit dem Doppelpunkt bekommen (vorher nicht, liegt es daran, dass ich auf alpine umgestellt hab?).
Verstehe aber auch nicht, warum hier der Doppelpunkt ist. Siehe:
https://stackoverflow.com/questions/16022624/examples-of-http-api-rate-limiting-http-response-headers